src/__tests__/request.test.js

```diff
@@ -59,6 +60,7 @@ describe('buildRequest', () => {
             maxContentLength: -1,
             method: 'post',
             params: null,
+            retries: 3,
```

Comment:
1 first call + 3 retries ot 1 try + 2 retires?

Reply:
Maybe I'll rename it "attempts".